src/creatures/players/player.hpp

```diff
@@ -88,9 +88,9 @@ struct OpenContainer {
 
 using MuteCountMap = std::map<uint32_t, uint32_t>;
 
-static constexpr int32_t PLAYER_MAX_SPEED = 65535;
-static constexpr int32_t PLAYER_MIN_SPEED = 10;
-static constexpr int32_t PLAYER_SOUND_HEALTH_CHANGE = 10;
+static constexpr uint16_t PLAYER_MAX_SPEED = std::numeric_limits<uint16_t>::max();
+static constexpr uint8_t PLAYER_MIN_SPEED = 10;
```

Comment:
I believe that here it should still be int32_t, since on a custom server the speed can end up being high, it is something that is up to whoever configures it.

Reply:
Also, minspeed should probably be of the same type as speed itself

Reply:
> Also, minspeed should probably be of the same type as speed itself

True